cli/src/cli_util.rs

```diff
@@ -2547,10 +2547,19 @@ pub struct EarlyArgs {
     #[arg(long, value_name = "WHEN", global = true)]
     pub color: Option<ColorChoice>,
     /// Disable the pager
-    #[arg(long, value_name = "WHEN", global = true, action = ArgAction::SetTrue)]
```

Comment:
unexpected change?

Reply:
Oh, that was me, I think I thought it was an obvious copypaste error I'm fixing at the time - not too sure now, but for color having the arg name be `when` - always/never/auto - makes sense, and for boolean no_pager having the arg name be `when` does not make sense, idk

But ofc feel free to split it into a whole separate PR or just remove it and forget it, I dont feel strongly

Reply:
I have reverted this fyi.